packages/shoreline/src/themes/sunrise/components/drawer.css

```diff
@@ -22,7 +22,7 @@
     --size: var(--size-medium);
   }
 
-  &[vaul-drawer-direction="right"] {
+  &[data-vaul-drawer-direction='right'] {
```

Comment:
@matheusps shouldn't this be double quoted just like the other ones from the sheet?